spec/features/user_views_best_tattoos_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'rails_helper'
+
+feature "User views the best tattoos", %(
+  As a user
+  I want to view the highest rated tattoos
+  So I know what submissions are the most awesome
+
+  Acceptance Criteria
+  [x] I must see the title and rating of each tattoo posting
+  [x] I must see only the 5 best tattoos
+) do
+
+  scenario 'visitor views best tattoos' do
+    10.times { FactoryGirl.create(:tattoo_with_reviews) }
+
+    t10 = Tattoo.find_by(title: "Celtic Armband 10")
+    t9 = Tattoo.find_by(title: "Celtic Armband 9")
+    t7 = Tattoo.find_by(title: "Celtic Armband 7")
+    t6 = Tattoo.find_by(title: "Celtic Armband 6")
+    t5 = Tattoo.find_by(title: "Celtic Armband 5")
+    t1 = Tattoo.find_by(title: "Celtic Armband 1")
+    Review.create(rating: 5, tattoo: t10, user: User.first)
+    Review.create(rating: 5, tattoo: t9, user: User.first)
+    Review.create(rating: 5, tattoo: t6, user: User.first)
+    Review.create(rating: 5, tattoo: t5, user: User.first)
+    Review.create(rating: 5, tattoo: t1, user: User.first)
+    Review.create(rating: 1, tattoo: t7, user: User.first)
```

Comment:
Also, you shouldn't need to specify a user here if you have your factory associations set up right!